.github/workflows/python-app.yml

```diff
@@ -0,0 +1,23 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python application
+
+on:
+  [push]
+jobs:
+  build:
+
+    runs-on: ubuntu-18.04
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install dependencies
+      run: |
+        python3 -m pip install --upgrade pip
+        pip3 install -r requirements-py3.txt
+        sudo add-apt-repository ppa:delmic-soft/odemis
+        sudo apt install python-pyro4-delmic
```

Comment:
Preferably, I'd go for *only* the use of apt, instead of a mix pip+apt. 

I wonder if "sudo add-apt-repository ppa:delmic-soft/odemis" has any chance to work, as usually it asks you to confirm, which you cannot. Either find a way to avoid the question, or use the "low level style" which consist in adding a line in `/etc/apt/sources.list` (followed by `apt update`).
Also, for the apt line, you probably have to add a "-y". I think it's also recommended to keep using "apt-get" in scripts, as the command line interface is more stable.